data_access.py

```diff
@@ -380,9 +380,20 @@ def __init__(self):
         self.all_occurrences[occurrence.document_id].append(occurrence.res_id)
 
     def get_all_documents(self):
+        """
+        :return: all documents in the database
+        """
         return list(get_all_documents())
 
     def create_resource(self, resource_name, data):
+        """
+        Create an entry into the database with date and user
+        :param resource_name: database table name
+        :type resource: str
+        :param data: column and value names of the table
+        :type data:dict
+        :return: JSON post with resource name and  dictionary
```

Comment:
It will actually return whatever the server sends back from the POST, usually a success or fail message.